packages/react/src/components/Tag/index.tsx

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames';
 interface TagProps {
   children: React.ReactNode;
   className?: string;
+  variant?: 'default' | 'small';
```

Comment:
Sorry if I didn't make this clear. One of the things we want to eventually do is define our properties better (as outlined on this issue https://github.com/dequelabs/cauldron/issues/1423).

Size properties are variants, but are not an aesthetic variant. As such they should have their own property `size` to not conflict with actual variants.